ios/ci.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+# This script runs from its own folder
+cd "$(dirname "$THIS_SCRIPT")"
+
+# Include our resource functions; they're pretty useful!
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+# Please note that this build script (understandably) assumes that it is running on Mac OS X.
+verify_on_mac
+
+builder_describe "Defines actions for use in CI-related iOS builds." \
+  "build" \
+  "prep-release"
+  # For consideration:  clean               ("Cleanup Carthage Mess", "Clean XCode DerivedData Mess")
+  # For consideration:  publish:testflight  ("Upload to TestFlight (PR area)")
+  # For consideration:  publish:app-store   ("Push * to the App Store for Deployment")
+  # For consideration:  publish:fastlane    (for matching release-build steps)
```

Comment:
app-store vs fastlane: fastlane is the tool for publishing to appstore/testflight so it doesn't make sense to publish to fastlane, no?